aider.el

```diff
@@ -149,7 +150,7 @@ If not in a git repository, an error is raised."
   (let ((git-repo-path (magit-toplevel)))
     (if (string-match-p "fatal" git-repo-path)
         (error "Not in a git repository")
-      (format "*aider:~%s*" git-repo-path))))
+      (format "*aider:%s*" git-repo-path))))
```

Comment:
also remove ~ character, cause it means home directory. here git-repo-path is the full directory instead.